src/lib/models/deployment.js

```diff
@@ -66,6 +73,29 @@ module.exports = class Deployment {
     })
   }
 
+  volumeMounts() {
+    return _.map(_.get(this.spec, 'volumes', []), volume => {
+      const out = {
```

Comment:
You can just return the object instead of assigning it and then returning it (I know that's how ports is; I could do better :-))

Reply:
Much better :)